Dockerfile.template

```diff
@@ -21,13 +21,13 @@ RUN	rm -rf /var/lib/mysql && \
 		| xargs -0 grep -lZE '^(bind-address|log|user\s)' \
 		| xargs -rt -0 sed -Ei 's/^(bind-address|log|user\s)/#&/' || true
 
-RUN mkdir /docker-entrypoint-initdb.d
-
 COPY docker-entrypoint.sh /usr/local/bin/
-RUN ln -s usr/local/bin/docker-entrypoint.sh / # backwards compat
 
-RUN chmod +x /usr/local/bin/docker-entrypoint.sh && \
-	chown -R mysql:mysql /usr/local/bin/docker-entrypoint.sh
+# symlink for backwards compatiblity
+RUN ln -s usr/local/bin/docker-entrypoint.sh / && \
+	chmod +x /usr/local/bin/docker-entrypoint.sh && \
+	chown -R mysql:mysql /usr/local/bin/docker-entrypoint.sh && \
+	mariadb --version | grep "${MARIADB_VERSION}"
```

Comment:
Adds a version test. This is something I'm trying to do, at minimum, for all images going forward. We don't want another PHP situation. :)

Reply:
Good idea 😆